wren-ai-service/CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Welcome to Wren AI Service contributing guide
+
+Thank you for investing your time in contributing to our project! This document provides guidelines for contributing to the Wren AI service.
+
+## New contributor guide
+
+- To get an overview of the project, please read the [concepts](README.md#concepts).
```

Comment:
what do you think about linking to the document site directly? it seems that sends me back the AI service README and I need to click the other link to redirect to the real concept page.

Reply:
ok